tools/crm_resource_runtime.c

```diff
@@ -667,8 +670,8 @@ cli_resource_delete(cib_t *cib_conn, crm_ipc_t * crmd_channel, const char *host_
     if (is_remote_node(node)) {
         attr_options |= attrd_opt_remote;
     }
-    rc = attrd_update_delegate(NULL, 'c', host_uname, rsc_name, NULL,
-                               NULL, NULL, NULL, NULL, attr_options);
+    rc = attrd_update_delegate(NULL, 'c', host_uname, rsc_name, operation,
+                               interval, NULL, NULL, NULL, attr_options);
```

Comment:
'interval' is the section and 'operation' the value?

not sure i like those being reused like that